modules/dashboard/terminal/picture.py

```diff
@@ -0,0 +1,54 @@
+import term, orientation, system, time, uos, json, machine
+
+system.serialWarning()
+term.header(True, "Picture")
+print("Loading...")
+
+pictures = []
+
+files = uos.listdir('/media')
+for f in files:
+	if f.endswith('.png'):
+		pictures.append("/media/"+f)
+
+try:
+	apps = uos.listdir('/lib')
+	for app in apps:
+		files = uos.listdir('/lib/'+app)
+		for f in files:
+			if f.endswith('.png'):
+				pictures.append("/lib/"+app+"/"+f)
+except:
+	pass
+
+current = machine.nvs_getstr('splash', 'logo')
```

Comment:
didi someone saw this does not work on your µPy version?

Reply:
correct that API has been replaced by a class `machine.NVS` or something like that. And we don't use nvs at all for any of our settings - and we don't have a config for setting a splashscreen either.

Reply:
Yeah, I missed this code as it's not used in the set of terminal apps we're using. Will adjust it.